.github/workflows/pip-release.yml

```diff
@@ -0,0 +1,147 @@
+# This file has been originally generated by maturin v0.14.17
+# To update, you can check
+#
+#    maturin generate-ci github --zig
+#
+# But note that some manual modification has been done.
+# Check the diffs to make sure that you haven't broken anything.
+
+name: pip-release
+
+on:
+  release:
+    types:
+      - "published"
+
+permissions:
+  contents: write
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64, x86, aarch64]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.7"
+      - name: Install dependencies
+        run: |
+          pip install patchelf --upgrade
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.target }}
+          args: --release --out dist --zig
+          sccache: "true"
+          manylinux: auto
+          working-directory: python
+      - name: Upload wheels
+        uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: python/dist
+      - name: Upload to release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: python/dist/*
+          tag: ${{ github.ref }}
+          file_glob: true
+
+  ### RustDDS does not support Windows
```

Comment:
Looks like the Windows build was fixed in https://github.com/jhelovuo/RustDDS/pull/290. Should I update our fork to include it?